src/libraries/System.ServiceProcess.ServiceController/src/System/ServiceProcess/ServiceController.cs

```diff
@@ -66,8 +70,12 @@ public ServiceController(string name, string machineName)
         }
 
 
+        /// <summary>
         /// Used by the GetServices and GetDevices methods. Avoids duplicating work by the static
         /// methods and our own GenerateInfo().
+        /// </summary>
+        /// <param name="machineName">Name of the machine</param>
```

Comment:
Nit, I think in general we don't doc private methods unless they're "non obvious". Just because it makes the code longer, and you're by definition already working within the type so you have some familiarity with it. I would remove param docs from private methods unless there's something really non obvious about them.